### DISCUSSION TITLE
Jobs are working, but Phases not showing

### DISCUSSION THREAD

**Discussion by jduraes** [swar/Swar-Chia-Plot-Manager#185] [General]
I have the manager running and the jobs are happening, completing successfully, but I cannot get the viewer to display information about phases, phase times, progress and temp_size.

![image](https://user-images.githubusercontent.com/5991582/117588014-bccfd280-b118-11eb-8060-26dfb7904b57.png)
(image is cropped on purpose)

The logs are being created successfully, with the lines correctly configured for them in config.yaml. I've got it running in a venv, and both the manager and viewer are running in the same venv.

What could be wrong here please?

Thanks.

**Comment by jduraes** [comment]
For reference, this is the view I get from another plotter viewer on the same jobs above:

![image](https://user-images.githubusercontent.com/5991582/117588134-6ca54000-b119-11eb-84e1-5b7646155491.png)

**Reply by 85256638** [reply]
I am having the same issue, what plotter viewer are you using?

**Reply by jduraes** [reply]
I'm using Swar's still, even with the issue as it only happens after a while.  

This other one (above) is static, so requires you to run it manually. https://www.powershellgallery.com/packages/PSChiaPlotter/1.0.14

**Reply by Hawstly** [reply]
I ran into the same issue when I was redirecting my Swar logs to a network share and once I editing my config and pointed the Swar logs to a local drive, the phases started showing correctly.

**Reply by jduraes** [reply]
Mine are on a local drive though. My final plots are stored on a remote
drive, but I doubt this would be the issue.

**Comment by jduraes** [comment]
I'm going to give this one a bump and ask, is no one else experiencing the same issue as I am? 

I mean, plot are working fine - it's just the viewer that does not display all the information that it should. Thanks.

**Comment by zeroarst** [comment]
I do have the same, did you solve it?

**Reply by jduraes** [reply]
For me, only a reboot "fixes" it. But it's also a temporary fix. After a while, the viewer goes back into the same state. We've been talking about it with Swar on Keybase, and he's not sure about what causes it, because he can't replicate it himself. We believe it may be something to do with the way psutils checks the logs, or when they were last touched. What I do now is after the problem happens, I stop the manager (viewer can continue, without progress info) and when the final plot ends, I reboot my machine and start again.

**Comment by kalosbg** [comment]
Start the view-er with in CMD with admin privileges or ensure it has rights to read the logs where they are located.

**Reply by jduraes** [reply]
Running as an Administrator does not change this behaviour. Plus yes, the user created and has access to those logs anyway.

**Comment by kalosbg** [comment]
Check the manager logs (in the folder where manager.py is located)

**Reply by jduraes** [reply]
No errors, I'm afraid. @swar has already mentioned in Keybase that he's not sure about what causes it, but thinks it's something to do with psutil interaction.
